.github/workflows/build_and_test.yml

```
@@ -14,6 +14,14 @@ on:
   pull_request:
     paths-ignore:
       - "**.md"
+  # Optionally use tmate to live debug failures:
+  # https://github.com/mxschmitt/action-tmate#manually-triggered-debug
+  workflow_dispatch:
+    inputs:
```

```suggestion
  workflow_dispatch:
    inputs:
      # Live debug failures using tmate by toggling input parameter 'debug_enabled':
      # https://github.com/mxschmitt/action-tmate#manually-triggered-debug
      debug_enabled:
```
Please put the comment on `debug_enabled` instead of the generic `workflow_dispatch`. I also suggested an alternative comment to help convey that you need to manually toggle the input parameter because that wasn't obvious to me at first, but feel free to change to anything that you think better conveys how to accomplish! Including maybe even a link directly to [Manually running a workflow](https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow) which IMO is more valuable then the link to the tmate documentation because that's more about doing what you did vs helping people that come across this code figure out how to run a workflow with `debug_enabled` toggled.

---

Applied, and added a link to manually running a workflow.